apps/pipelines/nodes/helpers.py

```diff
@@ -0,0 +1,22 @@
+from contextlib import contextmanager
+
+from django.db import transaction
+
+from apps.chat.models import Chat
+from apps.experiments.models import ConsentForm, Experiment, ExperimentSession
+from apps.users.models import CustomUser
+
+
+@contextmanager
+def temporary_session(team):
+    """A temporary sesssion setup that is rolled back after the context exits."""
+    with transaction.atomic():
+        owner = CustomUser.objects.create_user(username="test", password="test")
```

Comment:
what about using the current user?

Reply:
Sure. Added that [here](https://github.com/dimagi/open-chat-studio/pull/950/commits/432ec9c7a3c59d4a94162ab0a4b5845929a42a4c)